docs/developer_guide/contributing.md

```diff
@@ -0,0 +1,148 @@
+# Contributing
+
+Everyone is welcome to contribute, and we value everybody’s contribution. Code is thus not the only
+way to help the community. Answering questions, helping others, reaching out and improving the
+documentation are immensely valuable contributions as well.
+
+It also helps us if you spread the word: reference the library from blog posts on the awesome
+projects it made possible, shout out on Twitter every time it has helped you, or simply star the
+repo to say "thank you".
+
+Check out the official [ludwig docs](https://ludwig-ai.github.io/ludwig-docs/) to get oriented
```

Comment:
Could we please capitalize Ludwig in "ludwig docs"

Reply:
@Infernaught I cannot believe I missed that one.  Fixing immediately.